### ISSUE TITLE
A "cookbook" for common PDF uses would be nice for users

### ISSUE THREAD

**Issue by tbrowder** [pdf-raku/pdf-raku.github.io#2]
A practical cookbook would have an existing example program for each recipe. Some suggestions:

+ Given one or more paragraphs of text, output a PDF doc with paragraphs displaying various styles of justification

+ Given an existing PDF form, overwrite it with text and save as a new document

In addition, for each of the existing modules with example output, add beside it the program that generated each example.

**Comment by dwarring** [comment]
Yes, could add an link in examples from http://pdf-raku.github.io/

**Comment by dwarring** [comment]
Currently under development is [Pod::To::PDF](https://github.com/pdf-raku/Pod-To-PDF-raku), which is a PDF::API6 based Pod renderer.

Would be nice, if the cookbook could be published to both the web along with a PDF, which has the added benefit of dog-fooding.
